server/src/controller/questions.ts

```diff
@@ -1,5 +1,6 @@
 import type { models } from "../../../common";
 import { questionsRepository } from "../DAL";
+import { BadRequestError } from "../errors/BadRequestError";
```

Comment:
This should be used via index:

```suggestion
import { BadRequestError } from "../errors";
```